ASFHyP3/hyp3-cookiecutter
# Move to using a pyproject.toml and src layout

To test:

```bash
python -m pip install cookiecutter
cookiecutter https://github.com/ASFHyP3/hyp3-cookiecutter.git -c src_and_pyprojecttoml

{fill out form}

cd {hyp3 plugin name}

git init .
git add .
git commit -m "Minimal HyP3 plugin created with the hyp3 plugin cookiecutter"
git tag -a v0.0.0 -m "Marking zeroth release for auto-versioning and CI/CD Tooling"

python -m pip install .
```

Package should install successfully and behave like a normal python package.
